README.md

```diff
@@ -47,8 +47,10 @@ We use knex for migrations and seed data. Once your Docker environment is runnin
 
 - `npm run knex:migrate:latest`
   - Update your schema to the latest.
+- `npm run knex:migrate:down`
+  - Rollback the most single recent migration to your schema.
```

Comment:
I think this should be `Rollback the most recent single migration to your schema.`

Reply:
I'm unsure if it should be. I think that more says that your rolling back the most recent single migration you've run. I'm unsure quite how to explain it but it just sounds not quite right to me.

Reply:
hmm I sorta see what you are saying but I still think "Rollback the most single recent migration to your schema." could be improved - because it sounds like it is the "most single" which doesn't make sense. 

We should probably just use what the [knex docs](https://knexjs.org/guide/migrations.html) says, so I'd suggest "Undo the last migration that was run".

Reply:
Ah yeah that makes sense!